fl4health/clients/clipping_client.py

```diff
@@ -30,13 +31,15 @@ def __init__(
         device: torch.device,
         loss_meter_type: LossMeterType = LossMeterType.AVERAGE,
         checkpointer: Optional[TorchCheckpointer] = None,
+        warmed_up_module: Optional[WarmedUpModule] = None,
```

Comment:
Do you have to update the set_parameters in this example? It does not call the BasicClient set_parameters so I think we have to include the logic you did in BasicClient and DynamicWeightExchangeClient in the set_parameters for this class. I could be missing something though

Reply:
I changed the place of applying warmup model to initialize_all_model_weights function, so we would prevent from code duplication in files.